df15/index.html

```diff
@@ -100,6 +100,29 @@
 			</div>
 		</div>
 
+    <div class="row card" id="card_id">
+			<div class="col-lg-4 col-lg-offset-4 col-md-6 col-md-offset-3 col-xs-10 col-xs-offset-1">
+				<div class="row card_header" style="background-color: #A19B77;">
+          <h1>Dreamforce 2015</h1>
+          <h2>Lightning Components</h2>
+				</div>
+				<div class="row card_body">
+					<div class="challenge">
+            <h3>Challenge</h3>
+            <p>Create a simple master-detail application that displays the contacts for a selected account.</p>
+					</div>
+					<h3>Requirements</h3>
+					<ol>
+            <li>This challenge requires a Force.com Devleoper Edition: <a href="http://bit.ly/de15hack">bit.ly/de15hack</a>.</li>
```

Comment:
For consistancy, let's wrap the "Force.com Developer Edition" in the anchor tag instead of the ugly bit.ly link